src/ui/pages/salesPortal.page.ts

```diff
@@ -1,5 +1,78 @@
 import { PageHolder } from './pageHolder.page';
+import { logStep } from 'utils/reporter.utils';
 
-export abstract class SalesPortalPage extends PageHolder {
-  // Здесь разместить методы работы с хедером (задание для первого освободившегося)
+export class SalesPortalPage extends PageHolder {
+  // Основные контейнеры
+  readonly header = this.page.locator('header#main-header');
+
+  // Элементы хедера
+  readonly portalTitle = this.header.locator('span.fs-4');
+
+  // Навигационные элементы
+  readonly navItem = (name: string) => this.header.locator(`a[name="${name}"]`);
+  readonly activeNavItem = this.header.locator('a.active[name]');
+
+  // Элементы уведомлений
+  readonly notifications = {
+    button: this.header.locator('#notification-bell'),
+    badge: this.header.locator('#notification-badge'),
+    badgeText: this.header
+      .locator('#notification-badge')
+      .locator('xpath=./text()'),
+  };
+
+  // Элементы управления пользователем
+  readonly userControls = {
+    themeToggle: this.header.locator('#theme-toggle'),
+    profileLink: this.header.locator('#user-menu-button a'),
+    usernameText: this.header.locator('#user-menu-button a strong'),
+    signOutButton: this.header.locator('#signOut'),
+  };
+
+  uniqueElement = this.portalTitle;
+
+  @logStep('Navigate to {name}')
+  async navigateTo(
+    name: 'home' | 'orders' | 'products' | 'customers' | 'managers',
+  ) {
+    await this.navItem(name).click();
+  }
+
+  @logStep('Check if {name} is active')
+  async isNavItemActive(name: string): Promise<boolean> {
+    const classes = await this.navItem(name).getAttribute('class');
+    return classes?.includes('active') ?? false;
```

Comment:
В любом случае `includes` вернет `true | false`
```suggestion
    return classes?.includes('active');
```